_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,234 @@
+---
+id: a25f45
+name: headers attribute only refers to cells in the same table element.
+rule_type: atomic
+description: |
+  This rule checks that the headers attribute must only refer to cells in the same table element.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to `td` elements with a `headers` attribute, that are [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+## Expectation
+
+The `header` attribute of each target element is [a set of space separated IDs](https://www.w3.org/TR/html50/infrastructure.html#set-of-space-separated-tokens), each of which is an ID of an element that:
+1. has a [semantic role](#semantic-role) of `columnheader` or `rowheader`, and
+2. is a `cell` of the same [`table`](https://www.w3.org/TR/html50/tabular-data.html#concept-table) as the target element
```

Comment:
I don't think cell should be in backticks.
But you might want to link to the definition: [cell](https://www.w3.org/TR/html52/tabular-data.html#cell)